### PR TITLE
Update usage doc

### PR DESCRIPTION
**Pull request by Vincent056**
Add  remediation to compliance remediation section in the usage doc.

### CODE REVIEW

**Comment by Vincent056** [comment]
@mkumku Could you take a look on this when you have time? Thanks!

**Comment by mkumku** [comment]
Thank you. I think it looks better now.

File: doc/usage.md [hunk 1]
```diff
@@ -152,10 +152,12 @@ to `true`:
 $ oc edit -n $NAMESPACE complianceremediation/workers-scan-no-direct-root-logins
 ```
 
-The operator then creates a `MachineConfig` object per remediation. This
-`MachineConfig` object is rendered to a `MachinePool` and the
-`MachineConfigDeamon` running on nodes in that pool pushes the configuration
-to the nodes and reboots the nodes.
+The operator then creates a `MachineConfig` or `KubeletConfig` object per remediation. 
+This object is rendered to a `MachinePool` and the `MachineConfigDeamon` running on
```
Comment (mrogers950) [inline doc/usage.md:156]: Deamon -> Daemon :)

Comment (Vincent056) [reply doc/usage.md]: fixed! Thanks for the review

**Comment by jhrozek** [comment]
Since this is mostly an added link, I think it's safe to add the doc and qe labels out of band

**Pull request merged**